src/notifications/emails.py

```
@@ -105,15 +126,39 @@
             email (str): The recipient's email address.
             login_link (str): The login link to be included in the email.
         """
-        template = self._env.get_template(self._password_complete_email_template_name)
+        template = self._env.get_template(
+            self._password_complete_email_template_name
+        )
         html_content = template.render(email=email, login_link=login_link)
         subject = "Your Password Has Been Successfully Reset"
         await self._send_email(email, subject, html_content)
+
+    async def send_success_payment_email(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)